lib/sgtn-client/api/translation.rb

```diff
@@ -50,32 +42,26 @@ def self.getString_f(component, key, args, locale, *optionals)
     def self.getStrings(component, locale)
       SgtnClient.logger.debug "[Translation][getStrings]component=#{component}, locale=#{locale}"
       locale = SgtnClient::LocaleUtil.get_best_locale(locale)
-      items = get_cs(component, locale)
-      if (items.nil? || items['messages'].nil?) && !LocaleUtil.is_source_locale(locale)
-        items = get_cs(component, LocaleUtil.get_source_locale)
-      end
-
-      items
+      items = get_cs(component, locale)&.dig(:items)
+      items = items&.dig('messages').nil? && !LocaleUtil.is_source_locale(locale) ? get_cs(component, LocaleUtil.get_source_locale)&.dig(:items) : items
```

Comment:
This looks more clear.
```suggestion
      if items&.dig('messages').nil? && !LocaleUtil.is_source_locale(locale)
        get_cs(component, LocaleUtil.get_source_locale)&.dig(:items)
      else
        items
      end
```